src/pudl/transform/ferc1.py

```diff
@@ -824,11 +824,28 @@ class CalculationChecks(TransformParams):
     metrics_to_check: list[str] = [
         "error_frequency",
         "relative_error_magnitude",
-        "null_calculation_frequency",
+        "null_calculated_value_frequency",
         "null_reported_value_frequency",
     ]
-    group_checks: CalculationGroupChecks = CalculationGroupChecks()
-    metric_inputs: CalculationMetricInputs = CalculationMetricInputs()
+    group_metric_tolerances: GroupMetricTolerances = GroupMetricTolerances()
+    is_close_tolerance: CalculationIsCloseTolerance = CalculationIsCloseTolerance()
+
+    # @root_validator
+    # def grouped_tol_ge_ungrouped_tol(cls, values):
+    #     """Grouped tolerance should always be greater than or equal to ungrouped."""
+    #     group_metric_tolerances = values["group_metric_tolerances"]
+    #     groups_to_check = values["groups_to_check"]
+    #     for group in groups_to_check:
+    #         metric_tolerances = group_metric_tolerances.dict().get(group)
+    #         for metric_name, tolerance in metric_tolerances.items():
+    #             ungrouped_tolerance = group_metric_tolerances.dict()["ungrouped"].get(
+    #                 metric_name
+    #             )
+    #             if tolerance < ungrouped_tolerance:
+    #                 raise AssertionError(
+    #                     f"In {group=}, {tolerance=} for {metric_name} should be greater than {ungrouped_tolerance=}."
+    #                 )
+    #     return values
```

Comment:
Did this end up having other problems that weren't simple?

Reply:
The mechanics of the check are okay imo, but the substance of the check itself is a pain because of the various ways to set these tolerances. I think it would have been simpler if we could have removed one layer of defaults but as we discussed that was less simple.